openfisca_us/variables/irs/irs_gross_income.py

```diff
@@ -9,3 +9,12 @@ class irs_gross_income(Variable):
     documentation = "Gross income as defined in the Internal Revenue Code"
     definition_period = YEAR
     reference = "https://www.law.cornell.edu/uscode/text/26/61"
+
+    def formula(person, period, parameters):
+        COMPONENTS = [
```

Comment:
Minor but could be a parameter?

Reply:
Yeah or actually should I just use `c00100`?